clients/drcachesim/scheduler/scheduler.h

```diff
@@ -891,9 +890,10 @@ template <typename RecordType, typename ReaderType> class scheduler_tmpl_t {
         std::vector<input_ordinal_t> input_indices;
         int input_indices_index = 0;
         // Speculation support.
-        bool speculating = false;
+        std::stack<addr_t> speculation_split; // PC of resumption point.
```

Comment:
Why the "_split" suffix? I'd have expected speculation_stack.

Reply:
Because the stack is recording the split or divergence in path from the regular or prior-nested context: that's what is stored in the stack.  Maybe that's not clear.  I'm ok renaming to `speculation_stack`.  Done.